samples/nrf9160/fmfu_smp_svr/README.rst

```diff
@@ -0,0 +1,108 @@
+.. _fmfu_smp_svr_sample:
+
+Modem firmware update over SMP Server Sample
+############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The full modem firmware update sample implements a Simple Management Protocol (SMP) server.
+SMP is a basic transfer encoding that can be used with the MCUmgr management protocol.
+
+For more information about MCUmgr and SMP, see :ref:`device_mgmt`.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160ns
+
+.. include:: /includes/spm.txt
+
+Overview
+********
+
+The sample does the following:
+
+1. It deinitializes the :ref:`nrfxlib:nrf_modem`.
+#. It registers to MCUMgr a ``stat`` command first, then the commands to both upload the firmware and get the hash.
+#. It then enters an idle loop, waiting for any communication over the serial line.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/fmfu_smp_svr`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+Testing
+=======
+
+After programming the sample to your development kit, test it by performing the following steps:
+
+1. Connect the USB cable and power on or reset your nRF9160 DK.
+#. Open a terminal emulator, observe that the sample starts, and close the terminal emulator.
+#. Call the provided :file:`update_modem.py` script specifying COM port, firmware ZIP file, and UART baud rate.
+
+Sample Output
+=============
+
+The python script should give the following output:
+
+.. code-block:: console
+
+   # nrf9160 modem firmware upgrade over serial port example started.
+   {
+   "duration": 406,
+   "error_code": "Ok",
+   "operation": "open_uart",
+   "outcome": "success",
+   "progress_percentage": 100
+   }
+   Programming modem bootloader.
+   ...
+   Finished with file.
+   Verifying memory range 1 of 3
+   Verifying memory range 2 of 3
+   Verifying memory range 3 of 3
+   Verification success.
+   {
+   "duration": 5,
+   "error_code": "Ok",
+   "operation": "close_uart",
+   "outcome": "success",
+   "progress_percentage": 100
+   }
+   ------------------------------------------------------------
+
+Troubleshooting
+===============
+
+You can use the MCUMgr CLI tool as follows to test if the sample is running correctly:
+
+.. code-block:: console
+
+   mcumgr --conntype serial --connstring="dev=<COM Port>,baud=<Baudrate>" stat smp_com
+   stat group: smp_com
+       512 frame_max
+       504 pack_max
+
+Dependencies
+************
+
+This sample uses the following nRF Connect SDK libraries:
+
+* :ref:`mgmt_fmfu`
```

Comment:
Are these 2 ref names the right ones?

Reply:
This is what I'm trying to refrence https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/include/modem/modem_info.html?highlight=modem%20info